client/src/components/Tile.tsx

```diff
@@ -24,23 +25,33 @@ const Tile: React.FC<TileProps> = ({
   children,
   hide,
   itemToEquip,
+  tooltip,
 }) => {
   if (hide) return <></>;
 
   const tile = (
     <HStack px={2} textColor={disabled ? "gray.500" : undefined}>
       <TileImage imageUrl={imageUrl} imageAlt={imageAlt ?? header} />
       <VStack align="stretch" spacing={0.3}>
-        <Heading as="h3" size="sm">
-          {header}
-          {itemToEquip && <EquipLink itemToEquip={itemToEquip} />}
-        </Heading>
+        <HStack>
+          <Heading as="h3" size="sm">
+            {header}
+          </Heading>
+          {tooltip && tooltip}
+          <Text> {itemToEquip && <EquipLink itemToEquip={itemToEquip} />}</Text>
+        </HStack>
         {children}
       </VStack>
     </HStack>
   );
 
-  return href ? <MainLink href={href}>{tile}</MainLink> : tile;
+  return href ? (
```

Comment:
I was the one that snuck the box change in here.

I was messing with styling the tiles and noticed that various styling elements were applying differently depending on if they were wrapped the link or not. Wrapping them in boxes means each tile is the same at the highest level.

Reply:
HStack is effectively a Box (which is just a generic term for a display: block element), so I don't think there is a need for an additional div.

Reply:
![image](https://user-images.githubusercontent.com/44683462/150701956-585700bf-3187-454e-be3c-7b211699deb8.png)
My concern is that each tile is going to have different css styling applied at the highest level for the tile. Here I added a margin to the HStack and a border to show what was happening:
``` const tile = (
    <HStack px={2} textColor={disabled ? "gray.500" : undefined} border="8px" margin={2}>```